src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/AuditAndExceptionRepositoryServiceImpl.java

```diff
@@ -43,6 +47,7 @@ public void auditException(List<ExceptionCaseWorker> exceptionCaseWorkers) {
      *
      * @param exceptionCaseWorker ExceptionCaseWorker
      */
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
     public void auditException(ExceptionCaseWorker exceptionCaseWorker) {
         exceptionCaseWorkerRepository.save(exceptionCaseWorker);
     }
```

Comment:
can we not have a single transaction for  audit and exception instead of creating. a new one everytime?

Reply:
Will come back to you on this.

Reply:
@shashank-rastogi It is not possible as we are committing UP failures at the same time when we got hence it makes things complicated. and as we agreed  we are making db call to get exception we can't commit in end